src/Topic.hs

```diff
@@ -11,11 +11,13 @@ module Topic(
 , wrapVulDlr
 , stdWrap
 , Topic(..)
+, makeTopic
 ) where
 
 import Control.Monad.Trans.State.Strict(State)
 import System.Random(StdGen)
 
+import Output(Commentary, toCommentary, Showable)
```

Comment:
Keep the imports alphabetized

Reply:
This one already is alphabetized, but I should be on the lookout elsewhere! :sweat_smile: